test/validate-slack-request.js

```diff
@@ -6,9 +6,12 @@ var assert = require('assert')
 // Test object. Simulate an express request object
 var slackSigningSecret = '8f742231b10e8888abcd99yyyzzz85a5'
 
-function getTestHttpRequest (textArgs) {
-  return {
-    'X-Slack-Request-Timestamp': '1531420618',
+function getTestHttpRequest (textArgs, useGet=true) {
+  var req = {
+    'headers': {
+      'x-slack-request-timestamp': '1531420618',
+      'x-slack-signature': textArgs ? 'v0=a3e650d30d1e91901834f91d048c9d3c0a50e4dcffcef7bc67884e95df8588ce' : 'v0=a2114d57b48eac39b9ad189dd8316235a7b4a8d21a10bd27519666489c69b503',
+    },
```

Comment:
It would be cleaner to have two test files: the existing one (that test express requests) and a new one to test node's `http` requests. This would ensure that we don't introduce any regression adding the new feature